config/samples/end-to-end.yaml

```diff
@@ -10,48 +10,39 @@ metadata:
   name: end-to-end
   namespace: open-feature-demo
 spec:
-  featureFlagSpec: |
-    {
-      "flags": {
-        "new-welcome-message": {
-          "state": "ENABLED",
-          "variants": {
-            "on": true,
-            "off": false
-          },
-          "defaultVariant": "on"
-        },
-        "hex-color": {
-          "variants": {
-            "red": "CC0000",
-            "green": "00CC00",
-            "blue": "0000CC",
-            "yellow": "yellow"
-          },
-          "defaultVariant": "red",
-          "state": "ENABLED"
-        },
-        "fib-algo": {
-          "variants": {
-            "recursive": "recursive",
-            "memo": "memo",
-            "loop": "loop",
-            "binet": "binet"
-          },
-          "defaultVariant": "recursive",
-          "state": "ENABLED",
-          "targeting": {
-            "if": [
-              {
-                "in": ["@faas.com", {
-                  "var": ["email"]
-                }]
-              }, "binet", null
-            ]
-          }
+  featureFlagSpec:
+    flags:
+      new-welcome-message:
+        state: "ENABLED"
+        variants:
+          "on": true
+          "off": false
+        defaultVariant: "on"
+      hex-color:
+        state: "ENABLED"
+        variants:
+          "red": "CC0000"
+          "green": "00CC00"
+          "blue": "0000CC"
+          "yellow": "yellow"
```

Comment:
nitpick, but I don't think we need quotes on any of these fields, do we?

Reply:
Correct, I'll remove them.
Interestingly,
```
variants:
          "on": true
          "off": false
``` 
does require quotes otherwise on/off gets parsed as true/false.

Reply:
Unfortunately, a relic of an archaic version of yaml